Shared/UI/Views/TextView.swift

```diff
@@ -14,16 +14,18 @@ class TextView: UITextView {
 
     // Trim white space and newline characters when end editing. Default is true
     var trimWhiteSpaceWhenEndEditing: Bool = true
-
     // Maximum length of text. 0 means no limit.
-    var maxLength: Int = 250
+    var maxLength: Int = 500
+
+    @Published var isEditing: Bool = false
 
-    var cancellables = Set<AnyCancellable>()
+    private var cancellables = Set<AnyCancellable>()
 
     var numberOfLines: Int {
         guard let lineHeight = self.font?.lineHeight else { return 0 }
-        let height = self.contentSize.height - self.textContainerInset.top - self.textContainerInset.bottom
-        return Int(height / lineHeight)
```

Comment:
For whatever reason this line count logic was unreliable. The new logic seems to work really well.